### ISSUE TITLE
Switch: Extremely slow launching and loading

### ISSUE THREAD

**Issue by gingerbeardman** [libretro/RetroArch#12402]
I updated from 1.7.8 to 1.9.0 and then 1.9.2 latest. 

- Launching RetroArch now takes 80 seconds!
- Launching content (I'm playing Beetle VB right now) also takes takes 80 seconds

I've removed all cores I won't use, a trick needed on 3DS, but there has been no speed up. 

What is the cause of this slowness?

I plan to downgrade my RetroArch back to a version before this and I'll report back. 

Switch 10.2
CFW AMS 0.15.0

**Comment by gingerbeardman** [comment]
Problem solved!

- Delete the `cheats` folder

Now 2 seconds to launch RetroArch or any content.

**Comment by twinaphex** [comment]
hmm, @jdgleaver seems like what we talked about earlier with regards to finding a solution for the small cheat files being bad for performance on consoles is a more serious issue than imagined. Until we have a way of dealing with this I think it might be nest to strip out cheats from all console versions for now

**Comment by jdgleaver** [comment]
@twinaphex Cheat files are not included in any of the installation packages (for any platform) - they are only available via the online updater. Do you mean that the online updater cheats entry should be removed on console platforms?

**Comment by gingerbeardman** [comment]
I can't remember how and when I downloaded cheats on my setup, but I think the option should be disabled or come with a warning.

**Comment by twinaphex** [comment]
Yeah I guess if they are not shipped with the console ports, that is good. What we should do long-term is find a way to deal with this inherent slowness. I think the decision to ship each cheat file per game as a separate file might have been a mistake in the long run and we might want to think about cheat files that contain every game per system or something. We could then eventually migrate users over to that new system.
